jobs/settings.py

```diff
@@ -291,3 +291,13 @@
 }
 
 NOTIFICATIONS_ASYNC_QUEUE_NAME = CELERY_LOW_QUEUE_NAME
+
+
+# SMTP Setup
+
+EMAIL_BACKEND = env("EMAIL_BACKEND", default="django.core.mail.backends.console.EmailBackend")
+EMAIL_USE_TLS = env("EMAIL_USE_TLS", default=False)
```

Comment:
I think this should be `env.bool(`. 

Could be?

Reply:
Correct, I miss that one. Thanks :+1: